api-tests/healthcheck.py

```
@@ -1462,18 +1470,22 @@ def check_vendo_tagesbestpreis() -> str:
                                  "detail/share/split would break on it")
 
-    if best != 1:
-        raise CheckError(f"{best} intervals flagged istBestpreis (expected "
-                         "exactly 1) — the app marks what DB marks")
+    if best < 1:
+        raise CheckError(
+            "no interval flagged istBestpreis — "
+            "DB returned a tagesbestpreis calendar without a best-price interval"
+        )
     if not priced:
         raise CheckError("no interval carried angebotsPreis")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)